src/types/loans.ts

```diff
@@ -0,0 +1,38 @@
+import { MonetaryAmount } from "@interlay/monetary-js";
+import Big from "big.js";
+import { CurrencyExt } from "./currency";
+
+interface LoanReward{
+    currency: CurrencyExt;
+    apy: Big;
```

Comment:
Might be a good idea to indicate in a comment if the `apy` value is expected in percent or as a decimal rate

Reply:
Good idea, comment added - value will be in percentage.